custom_components/tuya_local/devices/electriq_sc16hpw_aircon.yaml

```diff
@@ -0,0 +1,216 @@
+name: ElectriQ SupremeCool SC16HPW Air Conditioner (XKA-G02)
```

Comment:
Branding belongs under a products section. This name should be generic in case of other products matching the same config.